hackclub.com.yaml

```diff
@@ -1245,6 +1245,9 @@ philanthropy:
   type: CNAME
   value: cname.vercel-dns.com.
 phs:
+  - ttl: 1
+    type: CNAME
```

Comment:
```suggestion
    type: ALIAS
```
try this

Reply:
(i know Vercel doesn't say to do this, but it's worked for us in the past)

Reply:
alright thanks, let us give it a try!

Reply:
rip didn't work